src/main/java/duke/Response.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+/**
+ * A duke.Response from the application to some user input.

```

Comment:
I think in your java docs, it might be easier to read and understand if you do not include your package in every the JavaDocs for every method.  Because it is clear from line 1 that this is part of the duke package. But the content in the JavaDocs are fine.